service.subtitles.legendastv/resources/lib/LTVutilities.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+
+import os
+import re
+import sys
+import xbmc
+import xbmcvfs
+import shutil
+import unicodedata
+import urllib, urllib2
+import ssl
+
+try: import simplejson as json
+except: import json
+
+try:
+    __scriptname__ = sys.modules[ "__main__" ].__scriptname__
+except:
+    __scriptname__ = ""
+
+TheTVDBApi = "A1738606AC58C23D"
+TMDBApi    = "96e0692265de9b2019b16f0c144efa56"
+
+def try_decode(text, encoding="utf-8"):
+    if isinstance(text, str):
+        try:
+            return text.decode(encoding)
+        except:
+            pass
+    return text
+
+def normalizeString(str):
+  return unicodedata.normalize(
+         'NFKD', unicode(unicode(str, 'utf-8'))
+         ).encode('ascii','ignore')
+
+def safeFilename(filename):
+  keepcharacters = (' ','.','_','-')
+  return "".join(c for c in filename if c.isalnum() or c in keepcharacters).rstrip()
+
+def log(msg, logtype="DEBUG"):
+  # xbmc.log((u"### [%s] - %s" % (__scriptname__,msg,)).encode('utf-8'),level=xbmc.LOGDEBUG )
+  if   logtype == "DEBUG":   loglevel = xbmc.LOGDEBUG
+  elif logtype == "NOTICE":  loglevel = xbmc.LOGNOTICE
+  elif logtype == "ERROR":   loglevel = xbmc.LOGERROR
+  reload(sys)
```

Comment:
can you change this to `xbmc.log(record.encode('utf-8', 'ignore')` in case of errors and `xbmc.log(record.encode('utf-8')` otherwise. Doesn't seem correct to change the default encoding through the sys module